src/stories/Iconography.stories.mdx

```diff
@@ -1,21 +1,70 @@
 <!-- Iconography.stories.mdx -->
```

Comment:
Per le icone conviene poi fare un bel ciclo for, perchè così è un po' verbose :)